src/App.jsx

```diff
@@ -183,9 +226,12 @@ function App() {
     };
     try {
       setIsSaving(true);
-      const resp = await fetch(url, options);
+      const resp = await fetch(
+        encodeUrl({ sortField, sortDirection, queryString }),
+        options
+      );
       if (!resp.ok) {
-        throw new Error('Failed editing todos');
+        throw new Error(`Error: ${resp.status} | Failed editing todos`);
       }
       const { records } = await resp.json();
       const savedTodo = {
```

Comment:
Similar code inside `completeTodo` let's consider what this is doing and is it the best approach? Keep in mind that in both instances of this code block, here in `updateTodo` and `completeTodo` nothing is being done with the mutated object